libreoffice-core/sw/source/uibase/uno/unotxdoc.cxx

```diff
@@ -178,26 +178,28 @@ using namespace ::com::sun::star::container;
 using namespace ::com::sun::star::document;
```

Comment:
Some auto format setting you have needs to be changed to exclude LibreOfficeKit.
The diff here should only include your changes.

Reply:
Reformated. There actually weren't any changes needed I had originally just had some extra info passed in but it wasn't necessary. It was just from playing around